tasks/ceph.py

```diff
@@ -558,7 +558,19 @@ def cluster(ctx, config):
                         stdout=StringIO(),
                         )
 
-                remote.run(args= ['yes', run.Raw('|')] + ['sudo'] + mkfs + [dev])
+                try:
+                    remote.run(
+                        args=['yes', run.Raw('|')] + ['sudo'] + mkfs + [dev]
+                    )
+                except run.CommandFailedError:
```

Comment:
I'm currently just reverting in the branch I'm using to test qa-suite
changes with anyway. If somebody else wants to push that into master so
that we aren't seeing failures in the nightlies that's probably wise, but
should take participation from somebody who's going to pay more attention
to than just whining that things aren't working. ;)